alacritty_terminal/src/ansi.rs

```diff
@@ -1483,15 +1502,74 @@ mod tests {
         static BYTES: &[u8] = &[0x1b, 0x5b, 0x31, 0x6d];
 
         let mut parser = Processor::new();
-        let mut handler = AttrHandler::default();
+        let mut handler = PseudoTermHandler::default();
 
         for byte in &BYTES[..] {
-            parser.advance(&mut handler, *byte, &mut Void);
+            parser.advance(&mut handler, *byte, &mut sink());
         }
 
         assert_eq!(handler.attr, Some(Attr::Bold));
     }
 
+    #[test]
+    fn parse_terminal_identity_csi() {
+        let bytes: &[u8] = &[0x1b, 0x5b, 0x31, 0x63];
```

Comment:
While I know what `0x1b` is and I think that's the best way to express that sequence in our codebase (since x1b is used everywhere), I have absolutely no clue what the other byte sequences are.

I'd be in favor of changing every readable byte to the byte char representation like this: `b'Z'`. That way I know immediately what we're testing, simply by looking at this input.